Remove old debug UI (#6478)
opentripplanner/OpenTripPlanner: doc/user/Frontends.md

On the other hand, **production frontends** are intended to be a component of larger public-facing deployments of OTP, with a more polished appearance and user experience. They generally do not work "out of the box", and require a significant amount of configuration and coordination with external components such as map tile servers and geocoders. They are designed to be components of a large professionally maintained and managed OTP deployment, and present a simpler view of OpenTripPlanner options and results to a non-technical audience of end users. 

## Debug Frontends

> The main OpenTripPlanner repository currently contains one debug web frontend: 

Reviewer: ```suggestion
```

Reviewer: I merged this into the next suggestion, see below

Author: I removed this line

> - the new frontend that is still being improved can be found at [`/client`](https://github.com/opentripplanner/OpenTripPlanner/tree/dev-2.x/client)
> - the classic one previously located at `/application/src/client/classic-debug/` has been removed

Reviewer: ```suggestion
OpenTripPlanner comes with one integrated debug web frontend. The frontend source code can be found at
[`/client`](https://github.com/opentripplanner/OpenTripPlanner/tree/dev-2.x/client).
```
We should aim for writing doc that is correct in 2 yers time. We do not need to include more than one paragraph about the old UI - that it is removed. That is done on line 13. In the rest of this doc we should fokus on the new UI.  There is no need to say "is still being improved", since we are constantly improving everything.

The **new debug client** is a React/TypeScript Single Page App (SPA) that can be served locally or accessed over a content delivery network (CDN). 
Unlike the original debug client, it connects to the OTP Java backend via the GraphQL API using the Transmodel vocabulary. By default, it is available at the root URL (`http://localhost:8080/` in local operation).

> The **classic debug client** was a jQuery and Backbone based UI whose history could be traced back over a decade to the first days of the OTP project. 
> It connected to the OTP Java backend via a REST API using the GTFS vocabulary. Historically this was the default OTP interface available at the root URL.
> It has been permanently removed.

Reviewer: ```suggestion
```

Author: I removed this paragraph
